test/cypress/e2e/moderationCreateHealthcareProvider.cy.ts

```diff
@@ -0,0 +1,107 @@
+import type { StateTree } from 'pinia'
+import enUS from '../../../i18n/locales/en.json'
+import fakeHealthcareProfessionalResult from '../../fake_data/moderation_dashboard/fakeModHealthcareProfessionalData.json'
+import { aliasQuery } from '../utils'
+
+let healthcareProfessionalsStore: StateTree
+
+before(() => {
+    cy.login()
+})
+
+describe('Moderation create healthcare professional form', () => {
+    context('Landscape mode', () => {
+        before(() => {
+            cy.intercept('POST', '**/', req => {
+                if (req.body.operationName === 'Query') {
+                    aliasQuery(req, 'HealthcareProfessionalSearchFilters', fakeHealthcareProfessionalResult)
+                }
+            })
+
+            cy.visit('/moderation')
+
+            cy.wait('@HealthcareProfessionalSearchFilters')
+
+            /* This will set the stores you need to access for testings.
+                    This NEEDS to be in the describe and not the login before*/
+            cy.window().then(win => {
+                healthcareProfessionalsStore = win.$pinia.state.value.healthcareProfessionalsStore
+            })
+
+            cy.get('[data-testid="submission-type-select"]').select('HEALTHCARE_PROFESSIONALS')
+            cy.get('[data-testid="add-hp-button"]').click()
```

Comment:
Nice work adding tests! 🙌🏾  Since `data-testid="add-hp-button"` is behind a conditional, we should also test that it doesn't exist if the conditions aren't met.